cob_base_velocity_smoother/mainpage.dox

```diff
@@ -2,9 +2,9 @@
 \mainpage
 \htmlinclude manifest.html
 
-\b cob_base_velocity_smoother is ... 
+\b cob_base_velocity_smoother is ...
```

Comment:
Those files are responsible for what you see e.g. [here](http://docs.ros.org/indigo/api/cob_base_velocity_smoother/html/)...
Thus, although they do not really help, I'd keep them...